src/java.base/share/classes/java/util/concurrent/LinkedTransferQueue.java

```diff
@@ -107,7 +108,11 @@ public class LinkedTransferQueue<E> extends AbstractQueue<E>
      * http://www.cs.rochester.edu/u/scott/papers/2009_Scherer_CACM_SSQ.pdf)
      * additionally arrange that threads enqueuing unmatched data also
      * block.  Dual Transfer Queues support all of these modes, as
-     * dictated by callers.
+     * dictated by callers. All enqueue/dequeue operations can be
+     * handled by a single method (here, "xfer") with parameters
+     * indicating whether to act as some form of offer, put, poll,
+     * take, or transfer (each possibly with timeout), as described
+     * bwlow.
```

Comment:
@DougLea Small typo—s/bwlow/below

Reply:
... which I just did, so hopefully easier to read through